tests/benchmark/config.json

```diff
@@ -5,64 +5,17 @@
       "params": {
         "conn_id": "bigquery",
         "metadata": {
-          "database": "astronomer-dag-authoring",
-          "schema": "tmp_astro"
+          "database": "bigquery"
         }
       }
     }
   ],
   "datasets": [
     {
-      "conn_id": "bigquery",
-      "file_type": "parquet",
-      "name": "ten_kb",
-      "path": "gs://astro-sdk/benchmark/trimmed/covid_overview/covid_overview_10kb.parquet",
-      "rows": 385817,
-      "size": "10 KB"
-    },
-    {
-      "conn_id": "bigquery",
-      "file_type": "csv",
-      "name": "hundred_kb",
-      "path": "gs://astro-sdk/benchmark/trimmed/tate_britain/artist_data_100kb.csv",
```

Comment:
Why is all this removed?

Reply:
Added missing test cases. I think this was more about facilitating quick testing.